studio/pages/api/auth/[ref]/config.ts

```diff
@@ -72,6 +72,9 @@ const handleGetAll = async (req: NextApiRequest, res: NextApiResponse) => {
     EXTERNAL_GOOGLE_ENABLED: false,
     EXTERNAL_GOOGLE_CLIENT_ID: null,
     EXTERNAL_GOOGLE_SECRET: null,
+    EXTERNAL_KEYCLOAK_ENABLED: false,
+    EXTERNAL_KEYCLOAK_CLIENT_ID: null,
+    EXTERNAL_KEYCLOAK_SECRET: null,
```

Comment:
I think we need to add a keycloak url and workos url field as well?

Reply:
Yup, good catch. Thanks!